tests-functional/tests/test_init_status_app.py

```diff
@@ -23,14 +23,14 @@ def test_init_app(self):
         backend_client.restore_account_and_login()
 
         assert backend_client is not None
+        backend_client.verify_json_schema(
+            backend_client.wait_for_login(), "signal_node_login")
```

Comment:
It's better to check login first, because it properly processes results of `restore_account_and_login` in case of an error. While other signals are just never sent in case of an error.